electricitymap/contrib/config/__init__.py

```diff
@@ -38,6 +45,26 @@
         if zone_name_1 not in ZONE_NEIGHBOURS:
             ZONE_NEIGHBOURS[zone_name_1] = set()
         ZONE_NEIGHBOURS[zone_name_1].add(zone_name_2)
+
+# Find neighbors to subzones and add them to parent zone.
+for zone in ZONES_CONFIG.keys():
+    subzones = ZONES_CONFIG[zone].get("subZoneNames", [])
+    if not subzones:
+        continue
+
+    for subzone in subzones:
+        for subzone_neighbor in ZONE_NEIGHBOURS[subzone]:
```

Comment:
isn't there a risk of iterating on `None`? 
Better to do `ZONE_NEIGHBOURS.get(subzone, [])`?

Reply:
Yes, I think you are right. I changed it